### PR TITLE
[rbdl-orb] Merging rbdl and rbdl-orb ports

### PR DESCRIPTION
**Pull request by ju6ge**
As of today https://github.com/rbdl/rbdl/pull/50 rbdl and rbdl-orb are now the same project again. This PR removes the now deprecated rbdl-orb port in favor of the rbdl port.
The rbdl port is updated to include everything rbdl-orb did.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
all :crossed_fingers:

### CODE REVIEW

File: ports/rbdl-orb/portfile.cmake [hunk 1]
```diff
@@ -1,45 +0,0 @@
-if (EXISTS "${CURRENT_INSTALLED_DIR}/share/rbdl/copyright")
```
Comment (LilyWangLL) [inline ports/rbdl-orb/portfile.cmake:1]: Please don't remove this port, just add remove the contents in **portfile.cmake** and add `set(VCPKG_POLICY_EMPTY_PACKAGE enabled)` in **portfile.cmake**. 
Then remove dependencies in vcpkg.json, and only add dependency `rbdl`. Please also change `port-version`.

Comment (LilyWangLL) [reply ports/rbdl-orb/portfile.cmake:1]: You can refer to port `openssl`.

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

File: ports/rbdl-orb/portfile.cmake (hunk 1 above)
Comment (LilyWangLL) [reply ports/rbdl-orb/portfile.cmake:1]: Ping @ju6ge for response.

Comment (ju6ge) [reply ports/rbdl-orb/portfile.cmake:1]: Sry for the delay. Was bussy with other stuff ;)

**Review by LilyWangLL — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**